index.html

```diff
@@ -5,8 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css">
 </head>
 <body>
-    
+    <header>
+        <h1>Weather Report</h1>
+        <h3>For the lovely city of</h3>
+        <h2 id="city-name">✨Seattle✨</h2>
```

Comment:
Heading tags (`<h1>`, `<h2>`, etc) should be used for setting up a table of contents, not for styling.  It makes more sense for all of these to be inside the `<h1>` and use other with classes to style them:

```suggestion
        <h1>Weather Report
        <em>For the lovely city of</em>
        <span id="city-name">✨Seattle✨<span></h1>
```